README.md

```diff
@@ -18,10 +18,10 @@ import (
 )
 
 func main() {
-  fdk.Do(myHandler)
+  fdk.Do(fdk.HandlerFunc(myHandler))
```

Comment:
Could we call it `fdk.Handle()` instead of `Do()`

Reply:
sure. `Serve` seems apt, as well. will go with Handle for now barring other feedback